src/components/graph/Graph.jsx

```diff
@@ -666,7 +689,7 @@ export default class Graph extends React.Component {
       this.state.config,
       this.state.highlightedNode,
       this.state.highlightedLink,
-      this.state.transform.k
+      this.state.transform
```

Comment:
Switching from passing around just the zoom to passing around the whole transform (it's needed later for the grid lines)